### ISSUE TITLE
Problem when changing source

### ISSUE THREAD

**Issue by Errante34** [Nimgoble/WPFTextBoxAutoComplete#16]
Hello, I'm having a problem with your textbox.

What happens is that when I set the source collection on the model for the first time, everything works great. 
**When I change it again**, the textbox **keeps working** and updates suggestions but does not highlight the suggested part of the text anymore: it's like if the enter key got pressed every time a suggestion triggers, **automatically accepting the suggestion**.

What i'm doing is nothing fancy:
```
<TextBox x:Name="inputTB" Height="45" Margin="10" FontSize="30" TextChanged="inputTB_TextChanged" Text="{Binding Path=InputText,Mode=TwoWay,UpdateSourceTrigger=PropertyChanged}"  
                 behaviors:AutoCompleteBehavior.AutoCompleteItemsSource="{Binding AutoCompleteList, UpdateSourceTrigger=PropertyChanged}"behaviors:AutoCompleteBehavior.AutoCompleteStringComparison="InvariantCultureIgnoreCase"
                 SelectionBrush="{Binding InputBoxSelectionColor}"
                 Background="{Binding InputBoxBackColor}" Foreground="{Binding InputBoxForeColor}"
           Dock="Top" VerticalContentAlignment="Center"/>
```

my model:
```
public List<string> AutoCompleteList
    {
        get { return _autoCompleteList; }
        set
        {
            if (value != _autoCompleteList)
            {
                _autoCompleteList = value;
                OnPropertyChanged(nameof(AutoCompleteList));
            }
        }
    }
```

**Comment by Nimgoble** [comment]
Hey @Errante34 ,

Could you upload an example project somewhere displaying this behavior?

**Comment by Errante34** [comment]
[Autocompletetest.zip](https://github.com/Nimgoble/WPFTextBoxAutoComplete/files/979220/Autocompletetest.zip)

Hello @Nimgoble ,

Here it is!
As you can see, when I change the source, autocompletition automatically accepts the suggested text

**Comment by Nimgoble** [comment]
Hey @Errante34 

This was my fault. It had been a while since I pushed a new version to nuget.  This has been fixed in 1.0.3.

**Comment by marcomaida** [comment]
oh well, fantastic then! =) Thanks
